server/service/individualServices/SoftwareUpgrade.js

```diff
@@ -100,9 +102,11 @@ async function PromptForBequeathingDataCausesTransferOfListOfApplications(user,
                         user,
                         xCorrelator,
                         traceIndicator + "." + traceIncrementer++,
-                        customerJourney
+                        customerJourney,
+                        undefined,
```

Comment:
in forwardRequest() function, after customerJourney it shall be directly isRequired. an undefined placeholder shall not hold here.